parser/go.mod

```diff
@@ -1,35 +0,0 @@
-module github.com/pingcap/tidb/parser
-
```

Comment:
Please not remove this. The parser is a public library for many developer to use. it is difficult from the other components in the tidb.

Reply:
I found failure in build pipeline 😭